### ISSUE TITLE
Cloud WS exception Error

### ISSUE THREAD

**Issue by andrefriasribeiro** [AlexxIT/SonoffLAN#869]
Hello!

I keep getting this error although everything seems to be working fine.
Could anyone provide me some insight where it came from?

```
2022-06-10 05:18:47 ERROR (MainThread) [custom_components.sonoff.core.ewelink.cloud] Cloud WS exception
Traceback (most recent call last):
File "/config/custom_components/sonoff/core/ewelink/cloud.py", line 315, in connect
if resp["error"] != 0:
KeyError: 'error'
2022-06-10 09:18:51 ERROR (MainThread) [custom_components.sonoff.core.ewelink.cloud] Cloud WS exception
Traceback (most recent call last):
File "/config/custom_components/sonoff/core/ewelink/cloud.py", line 315, in connect
if resp["error"] != 0:
KeyError: 'error'
2022-06-10 10:51:05 ERROR (MainThread) [custom_components.sonoff.core.ewelink.cloud] Cloud WS exception
Traceback (most recent call last):
File "/config/custom_components/sonoff/core/ewelink/cloud.py", line 315, in connect
if resp["error"] != 0:
KeyError: 'error'
2022-06-10 11:21:35 ERROR (MainThread) [custom_components.sonoff.core.ewelink.cloud] Cloud WS exception
Traceback (most recent call last):
File "/config/custom_components/sonoff/core/ewelink/cloud.py", line 315, in connect
if resp["error"] != 0:
KeyError: 'error'
2022-06-10 11:51:19 ERROR (MainThread) [custom_components.sonoff.core.ewelink.cloud] Cloud WS exception
Traceback (most recent call last):
File "/config/custom_components/sonoff/core/ewelink/cloud.py", line 315, in connect
if resp["error"] != 0:
KeyError: 'error'
```

**Comment by AlexxIT** [comment]
Please check latest release v3.1.0

**Comment by AlexxIT** [comment]
Fixed in v3.2.0
